src/renderer/components/modals/CreateMultisigAccount/components/WalletForm.tsx

```diff
@@ -66,6 +66,12 @@ export const WalletForm = ({
       threshold.value,
     );
 
+  const submitMstAccount: SubmitHandler<MultisigAccountForm> = ({ name, threshold }) => {
```

Comment:
Pass simpler payload so parent component won't know and handle `DropdownOption` for threshold